Api/Services/Crypto.cs

```diff
@@ -56,7 +56,7 @@ public string CreateToken(User user)
     {
         var claims = new List<Claim> {
             new Claim("Id", user.Id.ToString()),
-            new Claim(ClaimTypes.Name, user.Username),
+            new Claim(ClaimTypes.Name, user.Username +"," + user.Id.ToString()),
```

Comment:
```suggestion
            new Claim(ClaimTypes.Name, user.Username),
```
`Id` is **not** part of a name, it is found in the `Id` claim